Godeps/_workspace/src/github.com/openshift/openshift-sdn/plugins/osdn/subnets.go

```diff
@@ -130,6 +132,17 @@ func (oc *OvsController) SubnetStartNode(mtu uint) error {
 		oc.flowController.AddOFRules(s.NodeIP, s.SubnetCIDR, oc.localIP)
 	}
 
+	pods, err := oc.Registry.GetRunningPods(oc.hostName, kapi.NamespaceAll)
+	if err != nil {
+		return err
+	}
+	for _, p := range pods {
+		err = oc.pluginHooks.UpdatePod(p.Namespace, p.Name, kubetypes.DockerID(p.ContainerID))
+		if err != nil {
+			log.Warningf("Could not update pod %q (%s): %s", p.Name, p.ContainerID, err)
+		}
+	}
```

Comment:
We may have 40 to 100 pods per node and UpdatePod() on all them might be an expensive op, RunSDN()->StartNode()->SubnetStartNode() is called synchronously so this will have impact on openshift-node service bootup time. We only need to do this when cluster network is changed/br0 is recreated and this is relatively less frequent. So can we make oc.flowController.Setup() to tell if update pod is required or not and only perform this op if needed?